website/pages/community/index.jsx

```diff
@@ -2,10 +2,11 @@ import VerticalTextBlockList from '@hashicorp/react-vertical-text-block-list'
 import SectionHeader from '@hashicorp/react-section-header'
 import Head from 'next/head'
 import { productSlug } from 'data/metadata'
+import s from './style.module.css'
 
 export default function CommunityPage() {
   return (
-    <div id="p-community">
+    <div className={s.root}>
```

Comment:
Needed to remove deprecated `--site-max-width`, was straightforward to convert the few lines of CSS to CSS modules.